.vsts-ci.yml

```diff
@@ -53,7 +53,7 @@ jobs:
   # Maintenance #
   ###############
   - job: Maintenance
-    pool: lgb_agent_pool_ado
+    pool: lightgbm_agent_pool_ado
```

Comment:
context for this `pool:`  change: https://github.com/microsoft/LightGBM/issues/6949#issuecomment-3121444254